agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1725,7 +1725,9 @@ bool backhaul_manager::handle_slave_backhaul_message(std::shared_ptr<sRadioInfo>
 
         // If we're already connected, send a notification to the slave
         if (FSM_IS_IN_STATE(OPERATIONAL)) {
-            send_autoconfig_search_message(radio->front.iface_name);
+            m_task_pool.send_event(eTaskType::AP_AUTOCONFIGURATION,
+                                   ApAutoConfigurationTask::eEvent::START_AP_AUTOCONFIGURATION,
+                                   &radio->front.iface_name);
```

Comment:
you should consider at least pass event data as a const .

Reply:
Ok, it is no harm to do it.